src/features/Courses/AddClass/index.jsx

```diff
@@ -57,7 +57,7 @@ const AddClass = ({
         logError(error);
       } finally {
         if (isDetailClassPage) {
-          dispatch(fetchAllClassesData(selectedInstitution.id, courseInfo.masterCourseId));
+          dispatch(fetchClassesData(selectedInstitution.id, initialPage));
```

Comment:
In the class detail page the card detail data is not updated after the class is edited by this change.